doc/ref_model/chapters/chapter08.md

```diff
@@ -1,4 +1,320 @@
+The [Cloud Infrastructure Profiles](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4.2.4) section specifies two infrastructure profiles:
+
+The **Basic** cloud infrastructure profile is intended for use by both IT and Network Function workloads that have low to medium network throughput requirements.
+
+The **Network Intensive** cloud infrastructure profile is intended for use by applications that have high network throughput requirements (up to 50Gbps).
+
+The Network Intensive profile can specify extensions for hardware offloading; please see [Hardware Acceleration Abstraction](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#3.8). The Reference Model Network Intensive profile includes an initial set of [Network Intensive profile extensions](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#42421-network-acceleration-extensions).
+
+Based on the infrastructure deployed at the edge, the Table 8-3 specifies the [Infrastructure Profile features and requirements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md) that would need to be relaxed.
+
+**Table 8-3. TEC Exceptions to [Infrastructure Profile features and requirements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md)**
+
+<table>
+<tr>
+<th>Reference</th>
+<th>Feature</th>
+<th>Description</th>
+<th colspan="2">As Specified in RM Chapter 05 </th>
+<th colspan="2">Exception for Edge</th>
+</tr>
+<tr>
+<th></th>
+<th></th>
+<th></th>
+<th>Basic Type</td>        
+<th>Network Intensive</td>
+<th color="blue">Basic Type</td>        
+<th color="blue">Network Intensive</td>
+</tr>
+<tr><td>infra.stg.cfg.003</td><td>Storage with replication</td><td></td><td>N</td><td>Y</td><td>N</td><td>Optional</td></tr>
+<tr><<td>infra.stg.cfg.004</td><td>Storage with encryption</td><td></td><td>Y</td><td>Y</td><td>N</td><td>Optional</td></tr>
+<tr><<td>infra.hw.cpu.cfg.001</td><td>Minimum Number of CPU sockets</td><td>This determines the minimum number of CPU sockets within each host</td><td>2</td><td>2</td><td>1</td><td>1</td></tr>
+<tr><<td>infra.hw.cpu.cfg.002</td><td>Minimum Number of cores per CPU</td><td>This determines the number of cores needed per CPU.</td><td>20</td><td>20</td><td>2</td><td>2</td></tr>
+<tr><<td>infra.hw.cpu.cfg.003</td><td>NUMA alignment</td><td>NUMA alignment support and BIOS configured to enable NUMA</td><td>N</td><td>Y</td><td>N</td><td>Optional</td></tr>
+</table>
+
+<a name="8.3.4"></a>
+### 8.3.4  Telco Edge Cloud Infrastructure Characteristics
+This section characterises the hardware capabilities for different edge deployments and the Platform services that run on the infrastructure. Please note, that the Platform services are containerised to save resources, and benefit from intrinsic availability and auto-scaling capabilities.
+
+**Table 8-4. Characteristics of Infrastructure nodes
+
+<table>
```

Comment:
Please do not use raw html tables.

Reply:
@CsatariGergely This table needed to be in HTML because column spans

Reply:
@CsatariGergely replaced html with md - formatting not as good